module/actor/sheets/custos.js

```diff
@@ -9,17 +9,75 @@ import {LexArcana} from '../../config.js';
  */
 export default class LexArcanaCustosActorSheet extends LexArcanaActorSheet
 {
-    /**
-     * Define default rendering options for the NPC sheet
-     * @return {Object}
-     */
-    static get defaultOptions()
+  /**
+   * Define default rendering options for the NPC sheet
+   * @return {Object}
+   */
+  static get defaultOptions()
+  {
+    return mergeObject(super.defaultOptions,
     {
-        return mergeObject(super.defaultOptions,
-            {
-                classes: ["LexArcana", "sheet", "actor", LexArcana.ActorType.custos],
-                width: 720,
-                height: 680
-            });
+      classes: ["LexArcana", "sheet", "actor", LexArcana.ActorType.custos],
+      width: 720,
+      height: 680
+    });
+  }
+
+  /* -------------------------------------------- */
+
+  /** @override */
+  getData()
+  {
+    
+    // Basic data
+    let isOwner = this.entity.owner;
+    const data = {
+      owner: isOwner,
+      limited: this.entity.limited,
+      options: this.options,
+      editable: this.isEditable,
+      cssClass: isOwner ? "editable" : "locked",
+      isCharacter: this.entity.data.type === "custos",
+      isNPC: this.entity.data.type !== "custos",
```

Comment:
A custos may be a NPC given by the GM to the players to help them during the adventure. If those booleans serve the purpose of distinguishing the player characters and the NPCs controlled by the GM, I don't think this condition is going to work.

Reply:
Yeah that can't last, for the moment we have Custos and Friendly to distinguish them but clearly we can set a boolean there with a checkbox